### DISCUSSION TITLE
Multiplayer

### DISCUSSION THREAD

**Discussion by carolandreazza** [naverz/zepeto-studio-kor#564] [Issue Report (Legacy)]
Hi!
What is needed for more avatars to be able to see themselves within the world?
I already followed this guide 
https://github.com/naverz/zepeto-multiplay-example 
and still the avatars can't see each other

**Comment by Zepeto-TechEvangelist** [comment]
Hello~ Please follow the following instructions.

- All accounts to be tested must be registered as testers, and all must have ZEPETO friends.
- It should be updated to version 3.11.0 of the ZEPETO App.
- The room manager makes a room and invites you to invite your friend after entering. (Using Invite menu, not the invitation link)
- Out-of-room testers must be in the lobby and accept the invitation. (You must not be in another world)
